bindata/network/ovn-kubernetes/managed/alert-rules.yaml

```diff
@@ -0,0 +1,346 @@
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        count(count(min_over_time(ovn_db_cluster_id{db_name="OVN_Northbound"}[5m])) by (cluster_id)) > 1
+      for: 5m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesSouthboundDatabaseClusterIDError
+      annotations:
+        summary: Multiple OVN southbound database cluster IDs exist.
+        description: More than one OVN southbound database cluster ID indicates degraded OVN database high availability
+          and possible database split brain.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        count(count(min_over_time(ovn_db_cluster_id{db_name="OVN_Southbound"}[5m])) by (cluster_id)) > 1
+      for: 5m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesNorthboundDatabaseTermLag
+      annotations:
+        summary: OVN northbound databases RAFT term have not been equal for a period of time.
+        description: OVN northbound database(s) RAFT term have not been equal which may indicate degraded OVN database high availability.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        max(max_over_time(ovn_db_cluster_term{db_name="OVN_Northbound"}[5m])) - min(max_over_time(ovn_db_cluster_term{db_name="OVN_Northbound"}[5m])) > 0
+      for: 25m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesSouthboundDatabaseTermLag
+      annotations:
+        summary: OVN southbound databases RAFT term have not been equal for a period of time.
+        description: OVN southbound database(s) RAFT term have not been equal which may indicate degraded OVN database high availability.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        max(max_over_time(ovn_db_cluster_term{db_name="OVN_Southbound"}[5m])) - min(max_over_time(ovn_db_cluster_term{db_name="OVN_Southbound"}[5m])) > 0
+      for: 25m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesNorthboundDatabaseLeaderError
+      annotations:
+        summary: OVN northbound database(s) have no RAFT leader
+        description: OVN northbound database(s) have no RAFT leader. Networking control plane is degraded.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        count(max_over_time(ovn_db_cluster_server_role{db_name="OVN_Northbound", server_role="leader"}[5m])) == 0
+      labels:
+        severity: warning
```

Comment:
It should elect new leader immediately if the existing leader is in trouble. So it sounds like not a warning level.

Reply:
Agreed. Ill bump the severity and create a runbook.